packages/ui/src/components/Avatar/types.ts

```diff
@@ -14,12 +14,29 @@
  * limitations under the License.
  */
 
-import { Avatar } from '@base-ui-components/react/avatar';
-
 /** @public */
-export interface AvatarProps
-  extends React.ComponentPropsWithoutRef<typeof Avatar.Root> {
+export interface AvatarProps extends React.ComponentPropsWithoutRef<'div'> {
+  /**
+   * URL of the image to display
+   */
   src: string;
+
+  /**
+   * Name of the person - used for generating initials and accessibility labels
+   */
   name: string;
-  size?: 'small' | 'medium' | 'large';
+
+  /**
+   * Size of the avatar
+   * @defaultValue 'medium'
+   */
+  size?: 'x-small' | 'small' | 'medium' | 'large' | 'x-large';
```

Comment:
Are the more extreme ones here actually useful in practice? as opposed to falling back to exact values instead